hbase_storage/src/main/java/com/splicemachine/storage/ClientPartition.java

```diff
@@ -247,7 +247,8 @@ public void compact(boolean isMajor) throws IOException{
                     IOException ioe = exceptionFactory.processRemoteException(e);
                     if (ioe instanceof HNotServingRegion ||
                         ioe instanceof HWrongRegion ||
-                        ioe instanceof HRegionTooBusy) {
+                        ioe instanceof HRegionTooBusy ||
+                        e instanceof NullPointerException) {
```

Comment:
Since we use `e` here, we should pass `e` also in the debug message in the next line if it is an instace of NullPointerException.